Userland/Libraries/LibJS/Runtime/Temporal/Calendar.cpp

```diff
@@ -37,9 +37,10 @@ bool is_builtin_calendar(String const& identifier)
     // 1. Let calendars be AvailableCalendars().
     auto calendars = available_calendars();
 
-    // 2. If calendars contains id, return true.
-    if (calendars.contains_slow(identifier))
-        return true;
+    // 2. If calendars contains the ASCII-lowercase of id, return true.
+    for (auto calendar : calendars)
+        if (calendar.equals_ignoring_case(identifier))
+            return true;
```

Comment:
Please do use brackets for multiline things, even if they contain single line ifs
```cpp
for (auto calendar : calendars) {
	if (calendar.equals_ignoring_case(identifier))
    	return true;
}
```

Reply:
Makes sense, done